curriculum/challenges/english/01-responsive-web-design/css-picasso-painting/part-037.md

```diff
@@ -0,0 +1,160 @@
+---
+id: 60b69a66b6ddb80858c5159b
+title: Part 37
+challengeType: 0
+dashedName: part-37
+---
+
+# --description--
+
+Time to use some FontAwesome icons.
+
+The `i` element is used for idiomatic text, or text that is separate from the "normal" text content. This could be for italic text, such as scientific terms, or for icons like those provided by FontAwesome.
```

Comment:
```suggestion
The `i` element is used for idiomatic text, or text that is separate from the "normal" text content. This could be for _italic_ text, such as scientific terms, or for icons like those provided by FontAwesome.
```